### PR TITLE
e2e tests for Text component

### PR DESCRIPTION
**Pull request by olegrise**
@andrecardoso please review.

### CODE REVIEW

File: test/e2e/template-editor/cases/components/financial.js [hunk 1]
```diff
@@ -14,42 +15,37 @@ var FinancialComponentScenarios = function () {
...
-        helper.clickWhenClickable(templateEditorPage.getFinancialComponentEdit(), 'Financial Component Edit');
-        expect(templateEditorPage.getAddCurrenciesButton().isEnabled()).to.eventually.be.true;
-      }
-
       it('should show one Financial Component', function () {
-        // presentationsListPage.loadPresentation(presentationName);
-        _loadFinancialSelector();
-        expect(templateEditorPage.getInstrumentItems().count()).to.eventually.equal(3);
+        templateEditorPage.selectComponent("Financial - ");
+        // _loadFinancialSelector();
```
Comment (andrecardoso) [inline test/e2e/template-editor/cases/components/financial.js]: Can we remove this line?

File: test/e2e/template-editor/cases/components/financial.js [hunk 2]
```diff
@@ -66,9 +62,10 @@ var FinancialComponentScenarios = function () {
         expect(templateEditorPage.getSaveButton().isEnabled()).to.eventually.be.true;
 
         presentationsListPage.loadPresentation(presentationName);
-        _loadFinancialSelector();
+        // _loadFinancialSelector();
```
Comment (andrecardoso) [inline test/e2e/template-editor/cases/components/financial.js]: Same here

File: test/e2e/template-editor/cases/components/financial.js (hunk 1 above)
Comment (olegrise) [reply test/e2e/template-editor/cases/components/financial.js]: @andrecardoso good point. Will remove.

File: test/e2e/template-editor/cases/components/financial.js (hunk 2 above)
Comment (olegrise) [reply test/e2e/template-editor/cases/components/financial.js]: @andrecardoso thanks.

**Pull request merged**